matplotcheck/raster.py

```diff
@@ -140,25 +140,36 @@ def assert_legend_accuracy_classified_image(
         # Check that each legend entry label is in one of all_label_options
         assert len([val for val in legend_dict.values() if val]) == len(
             all_label_options
-        ), "Incorrect legend labels"
+        ), (
```

Comment:
let's rename this from
assert_legend_accuracy_classified_image
to 
assert_legend_labels 

i'm not sure why that other name was selected but it seems confusing!